gtk2_ardour/generic_pluginui.cc

```diff
@@ -565,7 +565,7 @@ GenericPluginUI::ControlUI::ControlUI (const Evoral::Parameter& p)
 	   below). be sure to include a descender.
 	*/
 
-	automate_button.set_sizing_text(_("Mgnual"));
```

Comment:
hm.. i don't understand why it needs a typo. could it be "the widest string you will put" instead?